### PR TITLE
zypper: add French translation

### PR DESCRIPTION
**Pull request by copolycube**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/linux/zypper.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`zypper refresh`
+
+- Installer un nouveau paquet :
+
+`zypper install {{package}}`
+
+- Supprimer un paquet :
+
+`zypper remove {{package}}`
```
Comment (kbdharun) [inline pages.fr/linux/zypper.md:16]:
```suggestion
`zypper remove {{paquet}}`
```

File: pages.fr/linux/zypper.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> SUSE & openSUSE utilitaire de gestion de paquets.
+> More information: <https://en.opensuse.org/SDB:Zypper_manual>.
+
+- Synchroniser la liste des paquets et versions disponibles :
+
+`zypper refresh`
+
+- Installer un nouveau paquet :
+
+`zypper install {{package}}`
```
Comment (kbdharun) [inline pages.fr/linux/zypper.md:12]:
```suggestion
`zypper install {{paquet}}`
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.fr/linux/zypper.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# zypper
+
+> SUSE & openSUSE utilitaire de gestion de paquets.
+> More information: <https://en.opensuse.org/SDB:Zypper_manual>.
```
Comment (kbdharun) [inline pages.fr/linux/zypper.md:4]:
```suggestion
> Plus d'informations : <https://en.opensuse.org/SDB:Zypper_manual>.
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**